src/validationinterface.h

```diff
@@ -87,13 +88,13 @@ class CValidationInterface {
      * but may not be called on every intermediate tip. If the latter behavior is desired,
      * subscribe to BlockConnected() instead.
      *
-     * Called on a background thread.
+     * Called on a background thread. Only called for the active chainstate.
```

Comment:
propbably not worth it to pass the chainstate role then?

Reply:
Good idea; done.